src/modules/services/reducers.js

```diff
@@ -87,18 +70,16 @@ export const services = (
       const items = action.data.filter((s) => s?.type);
       const itemsByID = Object.fromEntries(items.map((s) => [s.id, s]));
       const itemsByKind = Object.fromEntries(items.map((s) => [s.bento?.serviceKind ?? s.type.artifact, s]));
-      const itemsByArtifact = Object.fromEntries(items.map((s) => [s.type.artifact, s]));
 
       return {
         ...state,
 
         items,
         itemsByID,
         itemsByKind,
-        itemsByArtifact,
 
         // Backwards-compatibility with older Bento versions, where this was called 'federation'
```

Comment:
This comment no longer applies with these changes, right?

Reply:
yes, oops